### PR TITLE
MAT-4877 making multiple errors display

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-4877](https://jira.cms.gov/browse/MAT-4877)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

**Comment by sb-cecilialiu** [comment]
Pasted Graphic

File: src/components/statusHandler/StatusHandler.tsx [hunk 1]
```diff
@@ -2,23 +2,45 @@ import React from "react";
...
-            type="error"
-            content={
-              <h3 aria-live="polite" role="alert" data-testid={testDataId}>
-                {errorMessage}
-              </h3>
-            }
-            canClose={false}
-          />
-        );
+      if (errorMessages) {
```
Comment (RohitKandimalla) [inline src/components/statusHandler/StatusHandler.tsx]: Can these 2 conditions be grouped together, as for (!errorMessages) there is no else.

File: src/components/statusHandler/StatusHandler.tsx [hunk 2]
```diff
@@ -2,23 +2,45 @@ import React from "react";
...
+              }
+              canClose={false}
+            />
+          );
+        } else {
+          return <div />;
+        }
       }
+    } else {
+      return <div />;
```
Comment (RohitKandimalla) [inline src/components/statusHandler/StatusHandler.tsx]: Also is it possible to have a default return statement, as all our "else" block has the same content being returned?

**Review by RohitKandimalla — CHANGES_REQUESTED** [review]

File: src/components/statusHandler/StatusHandler.tsx (hunk 2 above)
Comment (sb-cecilialiu) [reply src/components/statusHandler/StatusHandler.tsx]: @RohitKandimalla Thanks a lot Rohit, just pushed in the change.

**Review by RohitKandimalla — APPROVED** [review]

**Pull request merged**